api/python/user/handlers.py

```diff
@@ -4,9 +4,13 @@
 
 @author: AilenZou
 '''
+import re
+
 import tornado.web
+
 import define
 from api.basehandler import BaseHandler, CustomHTTPError
```

Comment:
The BaseHandler.get_current_user will request the user service, which results in a infinite loop here.